flow360/component/simulation/operating_condition.py

```diff
@@ -87,29 +103,41 @@ class AerospaceCondition(Flow360BaseModel):
     atmosphere: ThermalState = ThermalState()
     reference_velocity_magnitude: Optional[VelocityType.Positive] = None
 
-    @validate_arguments
     @classmethod
+    @pd.validate_call
     def from_mach(
         cls,
-        mach: PositiveFloat,
+        mach: pd.PositiveFloat,
         alpha: float = 0,
         beta: float = 0,
         atmosphere: ThermalState = ThermalState(),
```

Comment:
is atmosphere a proper aerospace term here? I am not familiar

Reply:
This was Maciej's suggestion in one of the iterations. @maciej-flexcompute I think it makes sense but not 100% sure

Reply:
Ok then let's keep this for now.